server/models/Profile.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const profileSchema = new mongoose.Schema({
+  firstname: {
+    type: String,
+    required: true,
+  },
+  lastname: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  image: {
+    type: String, //to store image url
+    required: true,
+  },
+  availability: {
+    type: [{ date: Date, time: String }],
+    required: true,
+  },
+  rating: {
+    type: { Number, default: 0 },
+    max: 5,
+    required: true,
+  },
```

Comment:
What would be the plan with this? It would make more sense to have a reviews model and reference the profile and then calculate their overall score from each 5 star rating. Let's not focus on this though for the solo sprint.

Reply:
I was thinking of just adding this library [](https://www.npmjs.com/package/react-simple-star-rating) 
and  add method to the model to handle the avg rating